### ISSUE TITLE
Internal links doesn't work anymore in generated pdf

### ISSUE THREAD

**Issue by Andis59** [RickStrahl/MarkdownMonster#333]
If you use the TocGenerator Addin to create an Table of Content, the Link to an Id is not translated to a named destination in the pdf file. It creates an _file:///<path to where the source file is>#header_. In Adobe reader you then will get a dialog asking it it's OK to use this link! Which is then pointing to the Markdown file...

The problem seems to be that the _wkhtmltopdf_ option  **--enable-internal-links** (which is documented to be default) is not applied. If I add this option to the **FullExecutionCommand** then the named destinations are created and all is well!

**Comment by RickStrahl** [comment]
Hmmm... I added it to the default options that get set, but it's not working for me. I still get `file:///` links in the page to the TOC generated headers.

Looking around a bit I see a number of questions around this which may or may not have been fixed in **wkhtmltopdf**.

But maybe the more relevant thing is that the TOC uses `ID` attributes to route the link, and it might be that actual anchor tags are required:

https://stackoverflow.com/questions/20242054/wkhtmltopdf-internal-links

in which case the TOC generator has to be fixed to essentially embed the anchors.
